network-api/networkapi/templates/pages/buyersguide/product_page.html

```diff
@@ -173,7 +173,7 @@ <h2 class="tw-h3-heading">{% trans "Tips to protect yourself" %}</h2>
       {% with related_products=product.related_product_pages.all %}
         {% if related_products %}
           <div class="tw-col-span-full medium:tw-col-span-2 large:tw-col-start-2">
-            <div class="tw-flex tw-flex-col medium:tw-flex-row-reverse medium:tw-gap-5">
+            <div class="tw-flex tw-flex-col tw-items-baseline medium:tw-flex-row-reverse medium:tw-gap-5">
```

Comment:
Can you add some context why this is necessary? It looks like this is causing overflow on medium screens.

Reply:
Fixed